eng-projekt.html

```diff
@@ -303,13 +297,14 @@ <h1 style="font-size: 15px;font-family: 'Inter', sans-serif;font-weight: normal;
     </div>
 
 
-   <footer class="project blue stopka" id="kontakt">
-       <h2 style="margin-bottom: 0px; margin-top: 0px;">let's get in touch!<br/><a href="mailto:magdalenapiasecka40" style="color: black;">
-         magdalenapiasecka40@gmail.com</a></br> tel: +48 791 730 863</h2>
-     </footer>
+    <footer class="project blue footer_style" id="kontakt">
```

Comment:
Jeśli footer się nie zmienia to możemy go wyrzucić do osobnego pliku. Plusy są takie, że będziesz go tu tylko includowała i nie będziesz go zmieniała w x miejscach tylko w 1

Reply:
tutaj

Reply:
footer jest zmieniony (razem z href=tel) w development.html, czyli taki który można "kopiować" dalej